src/build.rs

```diff
@@ -0,0 +1,10 @@
+use std::fs;
+use std::io::Write;
+
+fn main() {
+    let json = include_str!("./../docs/assets/json/characters.json");
+    let data_string = ["json_parse!(", json,")"].concat();
+    let data_bytes = data_string.as_bytes();
+    let mut file = fs::File::create("./target/converted_data.rs").unwrap();

```

Comment:
[docs.rs](https://docs.rs/crate/galm/0.0.2/builds/244794) のビルドでエラーが発生している。

ディレクトリが見つからなくてエラーになってしまっているっぽい。

Reply:
## Build #244794 2020-04-29T18:23:01Z

```
$ rustc --version
rustc 1.45.0-nightly (647016923 2020-04-28)
$ cratesfyi --version
docsrs 0.6.0 (5cd4375 2020-04-29)
$ cratesfyi ...
[INFO] running `"docker" "create" "-v" "/home/cratesfyi/workspace/builds/galm-0.0.2/target:/opt/rustwide/target:rw,Z" "-v" "/home/cratesfyi/workspace/builds/galm-0.0.2/source:/opt/rustwide/workdir:ro,Z" "-v" "/home/cratesfyi/workspace/cargo-home:/opt/rustwide/cargo-home:ro,Z" "-v" "/home/cratesfyi/workspace/rustup-home:/opt/rustwide/rustup-home:ro,Z" "-e" "SOURCE_DIR=/opt/rustwide/workdir" "-e" "MAP_USER_ID=1001" "-e" "CARGO_TARGET_DIR=/opt/rustwide/target" "-e" "RUSTFLAGS=" "-e" "RUSTDOCFLAGS=-Z unstable-options --resource-suffix -20200428-1.45.0-nightly-647016923 --static-root-path / --cap-lints warn" "-e" "CARGO_HOME=/opt/rustwide/cargo-home" "-e" "RUSTUP_HOME=/opt/rustwide/rustup-home" "-w" "/opt/rustwide/workdir" "-m" "3221225472" "--cpus" "2" "--network" "none" "rustops/crates-build-env" "/opt/rustwide/cargo-home/bin/cargo" "+nightly" "doc" "--lib" "--no-deps" "-j2"`
[INFO] [stderr] WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
[INFO] [stdout] 775ec9f3ab5555c5694be69a8802ee8df328a8948b050f1491fc15e1759511cd
[INFO] running `"docker" "start" "-a" "775ec9f3ab5555c5694be69a8802ee8df328a8948b050f1491fc15e1759511cd"`
[INFO] [stderr]    Compiling galm v0.0.2 (/opt/rustwide/workdir)
[INFO] [stderr] error: failed to run custom build command for `galm v0.0.2 (/opt/rustwide/workdir)`
[INFO] [stderr] 
[INFO] [stderr] Caused by:
[INFO] [stderr]   process didn't exit successfully: `/opt/rustwide/target/debug/build/galm-c96759467694fec6/build-script-build` (exit code: 101)
[INFO] [stderr] --- stderr
[INFO] [stderr] thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/build.rs:8:20
[INFO] [stderr] note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[INFO] [stderr] 
[INFO] running `"docker" "inspect" "775ec9f3ab5555c5694be69a8802ee8df328a8948b050f1491fc15e1759511cd"`
[INFO] running `"docker" "rm" "-f" "775ec9f3ab5555c5694be69a8802ee8df328a8948b050f1491fc15e1759511cd"`
[INFO] [stdout] 775ec9f3ab5555c5694be69a8802ee8df328a8948b050f1491fc15e1759511cd
```

## Builds

rustc | docsrs | date
--- | --- | ---
rustc 1.45.0-nightly (647016923 2020-04-28) | docsrs 0.6.0 (5cd4375 2020-04-29) | 18 hours ago

## galm's sandbox limits

All the builds on docs.rs are executed inside a sandbox with limited resources. The limits for this crate are the following:

Available RAM | 3 GB
--- | ---
Maximum number of build targets | 10
Maximum rustdoc execution time | 15 minutes
Maximum size of a build log | 100 KB
Network access | blocked

If a build fails because it hit one of those limits please [open an issue](https://github.com/rust-lang/docs.rs/issues/new/choose) to get them increased.

Reply:
2c53804 で対応（したつもり）